Ogra/Encodable.swift

```diff
@@ -131,12 +131,8 @@ extension JSON {
 		case .Array(let array):  return array.map { $0.JSONObject() }
 		case .Object(let object):
 			var dict: [Swift.String : AnyObject] = [:]
-			for key in object.keys {
-				if let value = object[key] {
-					dict[key] = value.JSONObject()
-				} else {
-					dict[key] = NSNull()
-				}
+			for (key, value) in object {
+                dict[key] = value.JSONObject()
```

Comment:
I have a question about this. I don't think that the else condition will ever be reached since it would mean that there is no value for a key. This is impossible since `JSON.Object` has an associated value of `[String : JSON]` which has no optional values. Looping though all the keys will never be nil since they are of non optional type `String`. Do you have an example of an input that would yield a nil and that the else condition would be called?